.github/workflows/e2e-test-compatibility.yaml

```diff
@@ -0,0 +1,88 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release
+
+jobs:
+  e2e_test:
+    name: UI
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - uses: actions/setup-go@v1
+        with:
+          go-version: "1.13.5"
+      - name: Lookup yarn cache
+        id: yarn_cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn_cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-e2e-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-e2e
+      - name: Load go module cache
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Load TiUP cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.tiup/components
+          key: ${{ runner.os }}-tiup
+          restore-keys: |
+            ${{ runner.os }}-tiup
+      - name: Install and run TiUP in the background
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+          source /home/runner/.profile
+          tiup update playground
+          source /home/runner/.profile
+          tiup playground nightly --tiflash=0 &
```

Comment:
Maybe we can share some common scripts.

https://github.com/pingcap/tidb-dashboard/pull/1083/files#diff-4502f11bfc8852994bb2dccf25ef4ee07afd342f40de01feecf9b452f3ba7113

Reply:
Got it! I will take a look.